src/web/index.html

```diff
@@ -148,6 +148,7 @@
         }
 
         #default_menubar {
+            width: 100%;
```

Comment:
`menubar` is a block element (div) and therefore setting a width of 100% is redundant

Reply:
the setting is done because the body has the flex property in its css